<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>vlc</class>
 <widget class="QWidget" name="vlc">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>428</width>
    <height>465</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_19">
     <item>
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string>tuna.gui.vlc.source.name</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="cb_vlc_source_name"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>tuna.gui.vlc.scene</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="cb_scene"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>tuna.gui.vlc.source</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="cb_source"/>
     </item>
     <item>
      <widget class="QPushButton" name="btn_add_source">
       <property name="text">
        <string>tuna.gui.vlc.add</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QPushButton" name="btn_refresh_vlc">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>tuna.gui.vlc.refresh</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="btn_remove_source">
     <property name="text">
      <string>tuna.gui.vlc.remove.source</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
